__docs__/wonder-blocks-dropdown/multi-select.stories.tsx

```diff
@@ -252,6 +255,37 @@ CustomStylesOpened.parameters = {
     },
 };
 
+/**
+ * Here is an example of a dropdown that is in an invalid state.
+ */
+export const Invalid: StoryComponentType = {
+    render: Template,
+    args: {
+        isInvalid: true,
+    } as MultiSelectArgs,
```

Comment:
**thought:** How do you feel about using an example that uses a wrapper to manage state, where the `error` state is set `true` by default, and it's set `false` if the user chooses an option, so they can see how the Dropdown goes from `error` to valid in the example?

Reply:
that's a good idea! i've added the stories